src/components/@shared/AssetTeaser/index.tsx

```diff
@@ -51,13 +53,15 @@ export default function AssetTeaser({
             <Dotdotdot tagName="h1" clamp={3} className={styles.title}>
               {name.slice(0, 200)}
             </Dotdotdot>
-            {!noPublisher && <Publisher account={owner} minimal />}
+            {!noPublisher && <Publisher account={owner} />}
```

Comment:
The `minimal` prop on the `Publisher` component needs to be kept in this context though. It is dimming the the publisher output so they are not all pink on all asset teasers, which this current change did